lib/psql.js

```diff
@@ -305,8 +305,37 @@ var PSQL = function(connectionParams, poolParams, options) {
         return pg.Client.prototype.escapeLiteral(str);
     };
 
-    me.query = function(sql, params, callback, readonly) {
+
+    /**
+     * Executes an array of queries.
+     * Returns error if any query fails
+     * Returns the result of the last query (if successful)
+     */
+    function queryArray(client, sql, params, callback) {
+        client.query(sql[0], params, (err, result) => {
+            sql.shift();
+            if (err) {
+                return callback(err);
+            }
+            if (sql.length === 0) {
+                return callback(err, result);
+            }
+
+            return queryArray(client, sql, params, callback);
+        });
+    }
+
+    /**
+     *
+     * @param {String} sql - Query
+     * @param {Object} params - Query parameters
+     * @param {Function} callback - (error, result)
+     * @param {Boolean} readonly - If set, the query will be READ ONLY
+     * @param {Integer} timeout - Milliseconds to timeout the query
+     */
+    me.query = function(sql, params, callback, readonly, timeout) {
```

Comment:
Next iteration will be nice to refactor this API towards something like this:

```
me.query = function (options, callback) {
```
=>
```
me.query = function ({ sql, params, readonly, timeout }, callback) {
```
Thus, we can add more options without adding new parameters

Reply:
Actually, the future will look like:
```
async query ({ sql, params, readonly, timeout }) {
    return await [...];
}
```